client/state/jetpack/credentials/selectors.js

```diff
@@ -0,0 +1,9 @@
+import 'calypso/state/jetpack/init';
+
+export function getAreJetpackCredentialsInvalid( state, siteId, role ) {
+	return 'invalid' === state.jetpack?.credentials?.testStatus?.[ siteId ]?.[ role ];
```

Comment:
Since we already import the `jetpack` reducer on L1, `state.jetpack.credentials.testStatus` will always exist, so we don't need all the optional chaining here:

```suggestion
	return 'invalid' === state.jetpack.credentials.testStatus[ siteId ]?.[ role ];
```

Reply:
Fixed 👍